File: Baekjoon/BaaarkingDog/0x02_basic/2446.cpp
// 2446: 별 찍기 - 9

#include <iostream>

int main() {
  int N, step = 1;
  std::cin >> N;

  for (int i=0; i>-1; i+=step) {
    if (i == N-1) step = -1;

    // 좌측 공백
    for (int j=0;j<i; j++) std::cout << ' ';
    // 중앙 별
    for (int j=0; j<(N-i)*2 - 1; j++) std::cout << '*';

    std::cout << '\n';
  }

  return 0;
}

File: Baekjoon/BaaarkingDog/0x02_basic/2446.py
# 2446: 별 찍기 - 9

N = int(input())
step = 1
i = 0

while (i > -1):
  if (i == N-1): step = -1

  print(" " * i, end="")
  print("*" * ((N-i) * 2 -1))

  i += step